QUARKUS-3249.md

```diff
@@ -0,0 +1,32 @@
+# QUARKUS-3249 Configurable init container
+
+This document is a high level test plan for options to configure init containers in Quarkus 
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-3249
+
+## Scope of the testing
+* Check, that flyway task can be disabled
+* Check, that we can use custom image for the run and change its pull strategy
+  * I suppose, we should build the image ourselves, so a Docker file should be provided 
+
+### Current test coverage
+* Upstream coverage for Kubernetes, but without runs on the cluster
+
+### Impact on test suites and testing automation
+* Several new tests (probably, one or two new test classes) in `sql-db/panache-flyway` module
+
+### Impact on resources
+* No impact on maximum consumed cluster resources
+* Openshift JVM run will take five or ten more minutes for databases scenario, Openshift Native run will take twice as much
+  * Databases scenario is still shorter, than http-scenario, so no effect on total run time
+  
+## Future considerations
+* Check, that Quarkus uses better image (no image agreed in upstream yet) 
```

Comment:
If this is not agreed upon yet, do you think the feature is ready for RHBQ 3.8, strictly speaking? 3.8 is out on upstream. One of the main motivations was to not have CVE affected containers suddenly appearing in your cluster from Quarkus automagically (and that's still in the issue title), but I am not sure from the issue description if that goal of the feature request was yet met.

Reply:
On one hand, users now has an option to provide their own docker image for initialisation, so I would say it is safer, than before, since we do not force them to use a vulnerable image. On the other, I see that there is a plan to close this as wontfix. I asked for clarifications.